jetty-servlet/src/main/java/org/eclipse/jetty/servlet/ServletHandler.java

```diff
@@ -452,17 +455,18 @@ public void doScope(String target, Request baseRequest, HttpServletRequest reque
                 String servletPath = pathSpec.getPathMatch(target);
```

Comment:
Do we still need to get the pathSpec here?  If so, don't get it twice. get it once before the if.